lib/cypress/create_download_zip.rb

```diff
@@ -30,12 +30,24 @@ def self.all_patients
       file
     end
 
-    def self.create_total_test_zip(product, format = 'qrda')
-      measure_tests = MeasureTest.where(product_id: product.id)
+    def self.create_total_test_zip(product, format = 'qrda', criteria_list)
+      product_tests = ProductTest.where(product_id: product.id)
       file = Tempfile.new("all-patients-#{Time.now.to_i}")
       Zip::ZipOutputStream.open(file.path) do |z|
-        measure_tests.each do |m|
-          add_file_to_zip(z, "#{m.cms_id}_#{m.id}.#{format}.zip".tr(' ', '_'), m.patient_archive.read)
+        added_filter_test = false
+        product_tests.each do |pt|
+          if pt.class == MeasureTest
```

Comment:
Rather than comparing `class`, the Ruby way to do this is `pt.is_a?`. Alternatively, this control flow seems like it may be better suited to a `case` statement.